scene/gui/code_edit.cpp

```diff
@@ -927,8 +927,10 @@ void CodeEdit::_new_line(bool p_split_current_line, bool p_above) {
 		return;
 	}
 
-	const int cc = get_caret_column();
+	/* When not splitting the line, we need to factor in indentation from the end of the current line. */
```

Comment:
Use single-line comment syntax for single-line comments:

```suggestion
	// When not splitting the line, we need to factor in indentation from the end of the current line.
```

Remember to squash commits together after applying the suggestion (we don't use GitHub's *Squash and merge* feature because we want to keep the merge commit).

Reply:
Ah yeah that was a confusing one. The code_edit.cpp file has all of the single-line comments written in /* comment */ form. Would you still like me to change this one to use // single-line comment syntax? It might look a bit inconsistent with the rest of the file.

Reply:
> Would you still like me to change this one to use // single-line comment syntax? It might look a bit inconsistent with the rest of the file.

I guess it's not so important then; we can change it later in the whole fire in a single commit.